Lib/test/test_ast/snippets.py

```diff
@@ -599,4 +648,53 @@ def main():
 ('Expression', ('JoinedStr', (1, 0, 1, 8), [('FormattedValue', (1, 2, 1, 7), ('Name', (1, 3, 1, 4), 'a', ('Load',)), 114, None)])),
 ('Expression', ('JoinedStr', (1, 0, 1, 11), [('Constant', (1, 2, 1, 6), 'foo(', None), ('FormattedValue', (1, 6, 1, 9), ('Name', (1, 7, 1, 8), 'a', ('Load',)), -1, None), ('Constant', (1, 9, 1, 10), ')', None)])),
 ]
+
+# Lines below are generated by running Lib/test/test_ast/snippets.py -g_opt
```

Comment:
I still think that it would be better to generate these lines in two separated files.